src/test/groovy/com/intuit/graphql/orchestrator/batch/BatchLoaderExecutionHooksSpec.groovy

```diff
@@ -0,0 +1,23 @@
+package com.intuit.graphql.orchestrator.batch
+
+import graphql.ExecutionInput;
+import graphql.GraphQLContext;
+import helpers.BaseIntegrationTestSpecification
+
+class BatchLoaderExecutionHooksSpec extends BaseIntegrationTestSpecification {
```

Comment:
I think we can use `Specification` of groovy instead of `BaseIntegrationTestSpecification`